src/common/dom/setup-leaflet-map.ts

```diff
@@ -28,6 +28,14 @@ export const setupLeafletMap = async (
   style.setAttribute("rel", "stylesheet");
   mapElement.parentNode.appendChild(style);
   map.setView([52.3731339, 4.8903147], 13);
+
+  if (darkMode === undefined) {
+    darkMode =
```

Comment:
We should pass `hass` into the setup function instead of relying on a side effect of setting dark mode.

Reply:
So you want to add a property to `hass` if we are in dark mode?

Reply:
We need to store it somewhere, is it not in theme options ? Is there a way to deduct if a theme is "darkish"

Reply:
it is not just in theme options, as a user can also set a default dark theme, so the logic right now is:

default theme -> `dark mode option` or if not set `browser dark mode`
backend -> `browser dark mode`and `default dark theme` set

Reply:
So we would need `hass` and the result of the media query for dark mode.

Reply:
Can we store `darkMode: bool` in `hass.themes`?